test/Chirp.TestE2E/E2ETests.cs

```diff
@@ -695,6 +695,93 @@ public async Task TestShareCheepsVisibilityPrivateTimeline()
         // Clean up
         await DeleteUser();   
     }
+    [Test]
+    [Category("End2End")]
+    public async Task CheepingCheeps()
+    {
+        await RegisterUser();
+        await LoginUser();
+        
+        await _page.Locator("#CheepText").ClickAsync();
+        await _page.Locator("#CheepText").FillAsync("Hello World!");
+        await _page.GetByRole(AriaRole.Button, new() { Name = "Share" }).ClickAsync();
+        await Expect(_page.Locator("li").Filter(new() { HasText = "Hello World!" }).First).ToBeVisibleAsync();
+
+        
+        // Clean up and delete data
+        await DeleteUser();   
+    }
+
+    [Test]
+    [Category("End2End")]
+    public async Task CheepingFromPrivateTimeline()
+    {
+        {
+            await RegisterUser();
+            await LoginUser();
+            
+            await _page.GetByRole(AriaRole.Link, new() { Name = "My timeline" }).ClickAsync();
+            await _page.Locator("#CheepText").ClickAsync();
+            await _page.Locator("#CheepText").FillAsync("Hello World!");
+            await _page.GetByRole(AriaRole.Button, new() { Name = "Share" }).ClickAsync();
+            await Expect(_page.Locator("li").Filter(new() { HasText = "Hello World!" }).First).ToBeVisibleAsync();
+
+        
+            // Clean up and delete data
+            await DeleteUser();   
+        }
+    }
+    [Test]
+    [Category("End2End")]
+    public async Task EmptyCheeps()
+    {
+        {
+            await RegisterUser();
+            await LoginUser();
+            
```

Comment:
As mentioned before, make sure to go to public timeline: `await _page!.GotoAsync($"{AppUrl}");`